src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
+            if (!this.store.isCompact() && stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
Anyway, in the interest of getting this merged I'm happy to push a change to make it so that you can select anything in full view. Unless anyone else wants to weigh in?